libsolidity/formal/SMTEncoder.cpp

```diff
@@ -1427,13 +1429,36 @@ void SMTEncoder::bitwiseOperation(BinaryOperation const& _op)
 	optional<smtutil::Expression> result;
 	if (_op.getOperator() == Token::BitAnd)
 		result = bvLeft & bvRight;
-	// TODO implement the other operators
-	else
-		m_errorReporter.warning(
-			1093_error,
-			_op.location(),
-			"Assertion checker does not yet implement this bitwise operator."
-		);
```

Comment:
I think, with this PR, there are no more bitwise operators, so error is safe to be removed.